num_trees: 15
p_widht: 10
p_height: 10
initial_scale: 0.7
models: {1: 'barrel', 2: 'block'}
backgrounds: ['spacebot_bg']
forest_id: 'huiii'
database_path: '/home/stfn/recognition_database'